src/providers/tfc/runProvider.ts

```diff
@@ -44,6 +43,13 @@ export class RunTreeDataProvider implements vscode.TreeDataProvider<TFCRunTreeIt
         this.reporter.sendTelemetryEvent('tfc-runs-viewInBrowser');
         vscode.env.openExternal(run.websiteUri);
       }),
+      vscode.commands.registerCommand('terraform.cloud.run.viewPlan', async (run: PlanTreeItem) => {
```

Comment:
PlanTreeItem already stores the log URL, because by this point the user has expanded the Run to see the plan and/or apply status and we've queried for the plan info. We can either pass the entire PlanTreeItem object to the refresh method for PlanDataProvider, or pass the log url string instead of querying a second time inside PlanDataProvider

Reply:
Great catch, I'll fix that.